source/includes/kubernetes_extension/_k8_services.md

```diff
@@ -0,0 +1,120 @@
+### Services
+
+<!-------------------- LIST SERVICES -------------------->
+
+#### List services
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/services?cluster_id=a_cluster_id"
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "id": "alaintest-aerospike/default",
+      "ports": [
+        "3000/TCP",
+        "3002/TCP"
+      ],
+      "type": "ClusterIP",
+      "metadata": {},
+      "spec": {},
+      "status": {
+        "loadBalancer": {}
+      }
+    },
+    {
+      "id": "alertmanager-operated/monitoring",
+      "ports": [
+        "9093/TCP",
+        "9094/TCP",
+        "9094/UDP"
+      ],
+      "type": "NodePort",
+      "apiVersion": "v1",
+      "kind": "Service",
+      "metadata": {},
+      "spec": {},
+      "status": {
+       "loadBalancer": {}
+      }
+    }
+  ],
+    "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/services</code>
+
+Retrieve a list of all services in a given [environment](#administration-environments).
```

Comment:
You need to mention that the cluster id is a mandatory field here.

Reply:
Where?

Reply:
The same place as the other docs :p